src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -350,6 +353,52 @@ def load_emr_from_csmr_and_prevalence(key: str, location: str) -> pd.DataFrame:
     return data
 
 
+def load_cgf_exposure(key: str, location: str) -> pd.DataFrame:
+    key = EntityKey(key)
+    entity = utilities.get_entity(key)
+    location_id = (
+        utility_data.get_location_id(location) if isinstance(location, str) else location
+    )
+    data = get_exposure_without_model_version_id(entity, location_id)
+    data = reshape_to_vivarium_format(data, location)
+    return data
+
+
+def get_exposure_without_model_version_id(
```

Comment:
This is the same as the get_exposure function from vivarium_inputs but we drop the model version ID column. We do this because vivarium doesn't work with this value missing and some data transformations behave unexpectedly if we have more than one model version ID in the data.